Sources/Testing/Events/Recorder/Event.ConsoleOutputRecorder.swift

```diff
@@ -61,6 +61,12 @@ extension Event {
       public var useSFSymbols: Bool = false
 #endif
 
+      package init(useANSIEscapeCodes: Bool, ansiColorBitDepth: Int8, useSFSymbols: Bool) {
```

Comment:
You probably don't need a separate initializer for this and can just construct it property-by-property in `string(for:)` in the other file. [P3]

Reply:
This also helps fix the cross-platform build failures related to SF Symbols